core/wiki/macros/lingo.tid

```diff
@@ -15,8 +15,17 @@ $:/language/
 		<!-- First try the old school <active-lingo-base><title> format -->
 		<$transclude $tiddler={{{ [<active-lingo-base>addsuffix<title>] }}} $mode=<<parseMode>>>
 			<!-- If that didn't work, try the new <lingo-base><langcode>/<title> format -->
+			<!-- Get active language, if not language is selected, fallback to English -->
 			<$let language-code={{{  [[$:/language]get[text]get[name]else[en-GB]] }}}>
-        <$transclude $tiddler={{{ [<active-lingo-base>addsuffix<language-code>addsuffix[/]addsuffix<title>] }}} $mode=<<parseMode>>/>
+        <$transclude $tiddler={{{ [<active-lingo-base>addsuffix<language-code>addsuffix[/]addsuffix<title>] }}} $mode=<<parseMode>>>
+					<!-- Fallback to its first dependent, like "de-AT" will fallback to "de-DE" in some cases -->
+					<$let language-code={{{ [[$:/languages/]addsuffix<language-code>get[dependents]removeprefix[$:/languages/]first[]] }}}>
```

Comment:
The `{{{ filtered transclusion }}}` syntax always only returns the first element. So imo `first[]` is not needed